react/tic-tac-toe/src/app/screens/GameHistory/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+
+import styles from './styles.module.scss';
+
+import MatchesService from '~services/MatchesService';
+
+const Spinner = require('react-spinkit');
+
+class GameHistory extends Component {
+  state = {
+    isLoading: true,
+    history: [],
+    error: ''
+  };
+
+  getData() {
+    MatchesService.getMatches()
+      .then(response => {
```

Comment:
This logic should be placed inside the `getMatches` action.

Reply:
Ok! I take the advisement. This is going to change when i add recompose.